db/schema.rb

```diff
@@ -1217,6 +1232,8 @@
 
   add_foreign_key "active_storage_attachments", "active_storage_blobs", column: "blob_id"
   add_foreign_key "active_storage_variant_records", "active_storage_blobs", column: "blob_id"
+  add_foreign_key "companies", "accounts"
+  add_foreign_key "contacts", "companies"
```

Comment:
Once you rollback the migration and update it, this line should be removed automatically.